repoclosure/yum_f27.conf

```diff
@@ -0,0 +1,26 @@
+[main]
+cachedir=/var/cache/yum
+debuglevel=1
+reposdir=/dev/null
+logfile=/var/log/yum.log
+retries=20
+obsoletes=1
+gpgcheck=0
+assumeyes=1
+syslog_ident=repoclosure
+syslog_device=
+
+# repos
+[f27-base]
+name=BaseOS
+enabled=1
+mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=fedora-27&arch=x86_64
+failovermethod=priority
+
+[f27-puppet-5]
+name=f27-puppet-5
+baseurl=http://yum.puppetlabs.com/puppet5/fedora/27/$basearch/
```

Comment:
hardcoded arch and variable mixing looks odd

Reply:
You'd prefer using variables where possible? I can work that in.